twitter-borrowbot/twitterbotErrors.py

```diff
@@ -0,0 +1,123 @@
+class Error(Exception):
+    pass
+
+class TweepyAuthenticationError(Error):
+    def __init__(self, error=None):
+        self.error = error
+
+    def __str__(self):
+        return f"{type(self).__name__}: Failed to Authenticate with Twitter through Tweepy >> {self.error}"
+
+class FileIOError(Error):
+    def __init__(self, filename=None, data=None, error="Error not provided"):
+        self.filename = filename
+        self.data = data
+        self.error = error
+
+    def __str__(self):
+        if self.write:
+            return f"{type(self).__name__}: Failed to write '{self.data}' to file '{self.filename}' >> {self.error}"
+        return f"{type(self).__name__}: Failed to read from file '{self.filename}' >> {self.error}"
+
+    
+class LastSeenIDError(Error):
+    def __init__(self, filename=None, last_seen_id=None):
+        self.filename = filename
+        self.last_seen_id = last_seen_id
+
+    def __str__(self):
+        return f"{type(self).__name__}: The folowing ID '{self.last_seen_id}' from file '{self.filename}' is either too short or is not numeric"
+
+
+class GetMentionsError(Error):
+    def __init__(self, since=None, error=None):
+        self.since = since
+        self.error = error
+
+    def __str__(self):
+        return f"{type(self).__name__}: Tweepy failed to get the lastest mentions since '{self.since}' >> {self.error}"
```

Comment:
Codespell found this typo.
```suggestion
        return f"{type(self).__name__}: Tweepy failed to get the latest mentions since '{self.since}' >> {self.error}"
```